src/test/librbd/fsx.cc

```diff
@@ -2029,9 +2023,10 @@ main(int argc, char **argv)
 				usage();
 			break;
 		case 'l':
-			maxfilelen = getnum(optarg, &endp);
-			if (maxfilelen <= 0)
+			int _num = getnum(optarg, &endp);
```

Comment:
Well the fsx build fails because we're crossing initialization of that new variable (in the Jenkins build failure). :( Putting it in its own block would solve that problem, at least.